FreeRTOS-Plus/Source/FreeRTOS-Plus-TCP/FreeRTOS_ARP.c

```diff
@@ -261,6 +268,8 @@ uint8_t ucMinAgeFound = 0U;
 					optimisation. */
 					xARPCache[ x ].ucAge = ( uint8_t ) ipconfigMAX_ARP_AGE;
 					xARPCache[ x ].ucValid = ( uint8_t ) pdTRUE;
+
+					/* MISRA rule 15.5 relaxed for readability of code */
```

Comment:
How are you suppressing this violation?  I don't see a coverity annotation here.  Also, could this code be refactored?  For example, I think we'll also bump into the complexity > 8 issue in this function, it has a lot of branching.

Reply:
Reducing the branching will have to be done but you can do that in a separate PR.

Reply:
Sure thing 👍